lib/lightstep/span.rb

```diff
@@ -5,9 +5,12 @@ module LightStep
   #
   # See http://www.opentracing.io for more information.
   class Span
+    # Part of the OpenTracing API
+    attr_writer :operation_name
+
     # Internal use only
     # @private
-    attr_reader :guid, :trace_guid, :start_micros, :end_micros, :baggage, :tags
+    attr_reader :guid, :trace_guid, :start_micros, :end_micros, :baggage, :tags, :operation_name
```

Comment:
"I no speak Ruby good", but I wasn't able to make the `attr_reader` for `:operation_name` truly private a la the `attr_reader`s at the bottom of the class.